src/main/java/by/itechart/api/dto/UpdateUserDTO.java

```diff
@@ -0,0 +1,41 @@
+package by.itechart.api.dto;
+
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Data;
+
+@Data
+public class UpdateUserDTO {
+    @ApiModelProperty(
+            value = "User's first name",
+            name = "firstName",
+            dataType = "String",
+            example = "Adam")
+    String firstName;
+
+    @ApiModelProperty(
+            value = "User's last name",
+            name = "lastName",
+            dataType = "String",
+            example = "Smith")
+    String lastName;
+
+    @ApiModelProperty(
+            value = "User's email",
+            name = "email",
+            dataType = "String",
+            example = "adam.smith@gmail.com",
+            required = true)
```

Comment:
This will enforce update `email` each time user want to update some other data

Reply:
That's one of the reasons why inheritance can cause more problems than solves
You can try to resolve API documentation annotations on method (getter/setter) level